test/parallel/test-tls-connect.js

```diff
@@ -33,8 +34,8 @@ const tls = require('tls');
 // https://github.com/joyent/node/issues/1218
 // uncatchable exception on TLS connection error
 {
-  const cert = fs.readFileSync(path.join(common.fixturesDir, 'test_cert.pem'));
-  const key = fs.readFileSync(path.join(common.fixturesDir, 'test_key.pem'));
+  const cert = fs.readFileSync(path.join(fixtures, 'test_cert.pem'));
```

Comment:
This will fail tests as you cannot path.join fixtures. Moreover, this could be using method `fixtures.readSync` instead of `fs.readFileSync`.

Reply:
Hello @cassandradanger! Thank you for participating in the code and learn! This does have a few additional edits that will be necessary... specifically, the change here should be:

```js
const cert = fixtures.readSync('test_cert.pem');
const key = fixtures.readSync('test_key.pem');
```